matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/database/mapper/GroupSummaryMapper.kt

```diff
@@ -25,6 +25,7 @@ internal object GroupSummaryMapper {
     fun map(roomSummaryEntity: GroupSummaryEntity): GroupSummary {
```

Comment:
Change roomSummaryEntity to groupSummaryEntity

Reply:
done